paddle/majel/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
-cc_library(place SRCS place.cc)
-cc_library(ddim SRCS ddim.cc)
+cc_library(place SRCS place/place.cc)
+cc_library(ddim SRCS ddim/ddim.cc)
 
 if(WITH_TESTING)
```

Comment:
I created a new issue https://github.com/PaddlePaddle/Paddle/issues/2259 to record @Xreki 's suggestion.